wicket-jmx/src/main/java/org/apache/wicket/jmx/wrapper/RequestCycleSettings.java

```diff
@@ -117,9 +117,9 @@ public void setResponseRequestEncoding(final String responseRequestEncoding)
 	 * @see org.apache.wicket.jmx.RequestCycleSettingsMBean#setTimeout(java.lang.String)
 	 */
 	@Override
-	public void setTimeout(final String timeout)
+	public void setTimeout(final Duration timeout)
```

Comment:
I am not sure JMX will properly construct a Duration from the String input.

Reply:
@martin-g  I think you've lost me here :-). The old version of this method took in input a String and used a custom 'human-friendly' format to build a duration. For example something like '1.5 days' would have been converted into 36 hours. I've simply removed this textual format in order to use directly a standard Duration class.

Reply:
JMX works with a text-based protocol, like HTTP.
E.g. a user opens a UI and enters the values for the exposed settings. 
If the type of the property is a String or a primitive then all is fine (e.g. JMX will just use `Integer.parseInt(input)`). 
But if the of the property is of some complex type then someone has to help JMX to convert the String input to the target type.
I doubt there is a default converter for `java.time.Duration`. 

You can test it by starting a Wicket app and then connect to the JVM via JConsole or similar tool (e.g. wicketstuff-jmx-panel) and try to set a new value for `RequestCycleSettings#timeout`